### DISCUSSION TITLE
下载付费视频是不是需要先付费买过才能下载？

### DISCUSSION THREAD

**Discussion by freehehe** [lifegpc/bili#43] [Q&A] [answered]
PermissionError: [Errno 13] Permission denied: 'cookies.json'

**Comment by lifegpc** [comment, marked as answer]
这个错误是你没有权限访问本地文件，把文件夹移到别的无需管理员权限的位置。（Windows上移到非系统盘即可）
视频如果你能在网页上看，那一般都能下载，付费视频是需要付费的
